internal/errors.go

```diff
@@ -45,14 +45,26 @@ func TranslateError(err error, isCloudCluster bool, op ...string) (string, bool)
 
 func TranslateClusterError(err error, operation string) (string, bool) {
 	var urlError *url.Error
+	isRestApiEnabledError := `Cannot access Hazelcast REST API.
+REST service is disabled in the configuration by default. It enables you to manage cluster with rest-api calls.
+You should enable it on your CLUSTER MEMBERS to use the cluster commands.
+Check this link to find out more: https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#enabling-rest-api`
 	if errors.As(err, &urlError) && strings.Contains(urlError.Error(), "EOF") {
 		if operation == ClusterShutdown || operation == ClusterChangeState {
-			return "Cannot access Hazelcast REST API. Is it enabled? If yes, check CLUSTER_WRITE endpoint group is enabled https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#using-rest-endpoint-groups\nIf not check this link to find out more: https://docs.hazelcast.com/hazelcast/5.0/maintain-cluster/rest-api#enabling-rest-api", true
+			return `Cannot access Hazelcast REST endpoint.
+- Is REST API enabled?
+REST service is disabled in the configuration by default. It enables you to manage cluster with rest-api calls.
+You should enable it on your CLUSTER MEMBERS to use the cluster commands.
+Check this link to find out more: https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#enabling-rest-api
+
+- If yes, is CLUSTER_WRITE endpoint group enabled?
+Endpoints of Hazelcast REST API are grouped for fine-grained authorization. Commands such as "cluster change-state" that manipulates cluster state, must be enabled explicitly.
+Check this link to find out more: https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#using-rest-endpoint-groups`, true
```

Comment:
Isn't this text too long?

Reply:
Well sadly we cannot differentiate the underlying error. That's why we are printing 2 error messages. But I do not think they are too long individually. Do you?